quill-monix/src/main/scala/io/getquill/context/monix/MonixContext.scala

```diff
@@ -16,6 +16,4 @@ trait MonixContext[Idiom <: io.getquill.idiom.Idiom, Naming <: NamingStrategy] e
   // Need explicit return-type annotations due to scala/bug#8356. Otherwise macro system will not understand Result[Long]=Task[Long] etc...
   def executeQuery[T](sql: String, prepare: Prepare = identityPrepare, extractor: Extractor[T] = identityExtractor): Task[List[T]]
   def executeQuerySingle[T](sql: String, prepare: Prepare = identityPrepare, extractor: Extractor[T] = identityExtractor): Task[T]
-
-  protected val effect: Runner
```

Comment:
@juliano Regarding the `CassandraMonixContext`: Perhaps removing this was not ok? Then the `CassandraMonixContext` would also need to implement `effect` like it did before. But both were unused or I wouldn't have removed them.

Reply:
After reviewing again I don't think it's a problem 👍